index.html

```diff
@@ -1,16 +1,121 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+
+  <title>Great Idea!</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/styles.css">
 
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+  <div class="container">
+    <header class="header">
+      <nav class="nav-links">
+        <a href="services.html" class="nav-item">Services</a>
+        <a href="#" class="nav-item">Product</a>
+        <a href="#" class="nav-item">Vision</a>
+        <a href="#" class="nav-item">Features</a>
+        <a href="#" class="nav-item">About</a>
+        <a href="#" class="nav-item">Contact</a>
+      </nav>
+      <div class="nav-logo">
+        <a href="index.html">
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </a>
+      </div>
+    </header>
+
+    <section class="hero">
```

Comment:
Either I'll put the nav and the first section in the header or just the first section. But the first section can't be let alone between the `header` and `main`

Reply:
Okay got it,

Could the same logic be made for the footer, in that the section "contact" class should sit inside either the main or the footer element